arch/arm/boot/dts/overlays/revpi-connect-overlay.dts

```diff
@@ -85,21 +85,6 @@
 				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
-			eth2_int_pins {
-				brcm,pins     = <8>;
-				brcm,function = <BCM2835_FSEL_GPIO_IN>;
-				brcm,pull     = <BCM2835_PUD_OFF>;
-			};
-			eth2_reset_pins {
-				brcm,pins     = <40>;
-				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
-				brcm,pull     = <BCM2835_PUD_OFF>;
-			};
-			conbridge_int_pins {
```

Comment:
Add this to the pincontroller or move it to https://github.com/RevolutionPi/linux/blob/devel/revpi-5.10/arch/arm/boot/dts/overlays/revpi-con-can-overlay.dts as it is used there.